bin/plotting/pycbc_page_template_bin_table

```diff
@@ -56,11 +59,16 @@ columns = [bin_names, num_templates, frac_templates,
            num_triggers, frac_triggers, trigger_rate]
 columns = [np.array(c) for c in columns]
 
+format_strings = [None, '#', '#.###%', '#', '#.###%', '#.##E0']
```

Comment:
Seems OK, although I think one decimal place for '% of loud triggers' should be sufficient (ie `'#.#%'`)